openshift/templates/backup/backup-deploy.json

```diff
@@ -267,16 +289,16 @@
     {
       "name": "NAME",
       "displayName": "Name",
-      "description": "The name assigned to all of the resources defined in this template.",
+      "description": "The name assigned to all of the resources.  Use 'backup-postgres' for Postgres deployments or 'backup-mongo' for MongoDB deployments.",
       "required": true,
-      "value": "pgbackup"
+      "value": "backup-postgres"
     },
     {
       "name": "SOURCE_IMAGE_NAME",
       "displayName": "Source Image Name",
-      "description": "The name of the image to use for this resource.",
+      "description": "The name of the image to use for this resource.  Use 'backup-postgres' for Postgres deployments or 'backup-mongo' for MongoDB deployments.",
       "required": true,
-      "value": "pgbackup"
+      "value": "backup-postgres"
```

Comment:
Would it make sense to have these two settings coalesced into one, if the values are always going to be matched between them (to avoid errors).

Reply:
I’ve though about this for a bit.  I’d rather leave it the way it is.  I makes defining the SOURCE_IMAGE_NAME explicit rather than implicit and allows the end user name the deployment resources independently of the image name.

Reply:
Okay, that makes sense.